### PR TITLE
Synths Rebuilt (not gay ass lizard synths or whatever, like fallout synths get it fuck)

### PR DESCRIPTION
**Pull request by BobJoga**
## About The Pull Request

Basically balances synths so they can be a roundstart race, in a way. Synths are kind of funky so i'll have to do some weird fucking shit to make this work.
Basically, what i did is make it a fucking quirk - This may seem weird, but it is way easier for me since the synth species inherits shit from another species on being gained.
This also helps in one way, as you'll be able to tell who is a synth with a health analyzer by the quirk - but not just by normally examining.

## Why It's Good For The Game

I know for a fact there is some rando who roleplays as a synth (angel rainwater i think), so there is at least some demand for it
its synths they're the funny fallout 4 race or whatever
more roleplay opportunities yada yada
basically expensive IPCs i guess(???)
prettyhuman

aight bro imma head out
aightbro

## Changelog
:cl:
add: Added a synth neutral quirk... which turns you into a synth once you spawn. Upsides and downsides are detailed in the desc. Fallout jokes yada yada yeah.
balance: Synths have been rebalanced for the synth quirk. Now they can be dismembered and their limbs can be disabled, and they breathe.
refactor: Synth species datum has been changed a bit so custom limb icons are taken into account etc.
/:cl:

### CODE REVIEW

**Comment by BobJoga** [comment]
Alright this is way funkier than i thought, i'll have to do some STUFF to make it work

**Comment by BobJoga** [comment]
fuck it i'll just take the WIP tag off it's mostly working fuckdfgh

**Comment by improvedname** [comment]
I really really dislike this

**Comment by ShadeAware** [comment]
Why a quirk? This seems like something that should be a race entirely.

**Comment by BobJoga** [comment]
> Why a quirk? This seems like something that should be a race entirely.



> I really really dislike this

I have explained the reasoning for the quirk in the fucking PR just read it you blind or what

**Comment by BobJoga** [comment]
**PR's closed due to mutes caused by Kathrin Bailey. For any grievances or complaints, contact Floof Ball#0798 on discord. The PR owner will also not listen to any of your complaints unless you provide screenshots that prove you have contacted Floof Ball#0798 on discord first.**
pool
